CMakeLists.txt

```diff
@@ -189,13 +189,13 @@ if(SPHINX_FOUND)
   )
 
   set(SPHINX_SOURCE ${PROJECT_SOURCE_DIR}/docs)
-  set(SPHINX_BUILD ${PROJECT_BINARY_DIR}/docs/)
+  set(SPHINX_BUILD ${PROJECT_SOURCE_DIR}/docs/_build)
```

Comment:
@alexsavulescu : without understanding all details, isn't there a way to use BINARY dir instead of SOURCE dir? Typically we want to keep source directory tree and hence avoid generating anything inside main source directory.

Reply:
That was the original idea. But now since RTD is in the picture, we need to be consistent. That is the regular sphinx output directory.

Reply:
Hmm...Ok. If there is no other easy way, lets go with this for now.